saltproc/depcode.py

```diff
@@ -595,9 +603,9 @@ def write_depcode_input(
         Parameters
         ----------
         template_file : str
-            Path to user template file for depletion code..
+            Path to Serpent2 template file.
         input_file : str
-            Path to input file for depletion code rerunning.
+            Path to input file for Serpen2 rerunning.
```

Comment:
"Serpent2", I think you're missing a 't'.

Reply:
Good catch! Fixed